### PR TITLE
Fixes #25460 - only set the bind_host to :: if IPv6 is available

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
and I have no idea why the tests fail.

**Comment by ekohl** [comment]
Restarted the failing test just to be sure. Should be fixed now with the updated puppetlabs-apt.

**Comment by evgeni** [comment]
@ekohl like this?

File: spec/classes/foreman_proxy__config__spec.rb [hunk 1]
```diff
@@ -99,24 +93,60 @@
...
+              ':https_port: 8443',
+              ':log_file: /var/log/foreman-proxy/proxy.log',
+              ':log_level: INFO',
+              ':log_buffer: 2000',
+              ':log_buffer_errors: 1000',
+            ])
+          end
+        end
+
+        context 'without IPv6' do
```
[the remark this answers is not in the document]
Comment (evgeni) [reply spec/classes/foreman_proxy__config__spec.rb]: right, updated

**Pull request merged**